### ISSUE TITLE
Latest release causes: TypeError: Cannot read property 'group' of undefined

### ISSUE THREAD

**Issue by Swiftnesses** [Foddy/node-red-contrib-huemagic#139]
Trying to set a scene causes:

```
TypeError: Cannot read property 'group' of undefined
    at HueScene._inputCallback (/config/node-red/node_modules/node-red-contrib-huemagic/huemagic/hue-scene.js:28:38)
    at HueScene.Node._emitInput (/opt/node_modules/@node-red/runtime/lib/nodes/Node.js:199:18)
    at Immediate._onImmediate (/opt/node_modules/@node-red/runtime/lib/nodes/Node.js:179:33)
    at processImmediate (internal/timers.js:439:21)
```

This worked perfectly until the last release.

**Comment by Foddy** [comment]
Fixed with 2.6.1+

**Comment by Swiftnesses** [comment]
Thank you @Foddy :-)
